lstmcpipe/onsite_mc_r0_to_dl3.py

```diff
@@ -100,33 +107,30 @@
     # Create log files
     log_file, debug_file, scancel_file = create_log_files(prod_id)
 
+    # Make sure the lstchain config is defined if needed
+    # It is not exactly required if you process only up to dl1
+    if any([step not in ('r0_to_dl1', 'merge_and_copy_dl1') for step in stages_to_run]):
+        assert abspath(args.config_file_lst), 'The lstchain config needs to be defined for all steps following dl1 processing'
```

Comment:
I'm not sure that using `assert` statements in scripts that are not a tests is a good practice. @vuillaut or @maxnoe can comment on this for sure :)

Maybe an `if` to check if the argument is passed correctly, or an `os.path.isfile()` or an `os.path.exists()` could work

Reply:
Sure, we can raise an Exception as well

Reply:
> Maybe an `if` to check if the argument is passed correctly, or an `os.path.isfile()` or an `os.path.exists()` could work

These will raise
`TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType'`
which I though was not descriptive enough.
It just checks if the variable is set, because it is not required by argparse anymore.
You could just do
```python
if args.config_file_lst is None:
    raise ...
```